include/PathUtil.h

```diff
@@ -48,12 +48,14 @@ namespace lmms::PathUtil
 	QString LMMS_EXPORT basePrefix(const Base base);
 	//! Check the prefix of a path and return the base it corresponds to
 	//! Defaults to Base::Absolute
-	Base LMMS_EXPORT baseLookup(const QString & path);
+	Base LMMS_EXPORT baseLookup(const QString & input);
 
+	//! Enforce forward slash (/) for folder paths for uniform handling.
+	QString LMMS_EXPORT serializePath(const QString & input);
```

Comment:
Looks like the new function is not used outside of `PathUtil`? If it's an implementation detail that isn't intended to be used by other programmers, it should be in `PathUtil.cpp` only, inside an anonymous namespace.

Reply:
This was completed as part of 252e1a2.